ISISComputingGroup/EPICS-inst_servers
# Ticket 5413: Cset not call blockserver

### Description of work

See https://github.com/ISISComputingGroup/IBEX/issues/5413

cset will now assume that blocks are uppercase and not ask blockserver for block names.

This means that:
* The gateway needs to have aliases for both the upper and lowercase for setting the PV
* Runcontrol needs to have aliases for both the lower and uppercase (this was done by moving the runcontrol IOC into a different namespace and getting the gateway to create aliases for it)

### To test

* Create a block that's mixed case e.g. MyBlock pointing to a PV that you can change
* Confirm that you can do `caput %MYPVPREFIX%CS:SB:MyBlock:SP 1` and `caget %MYPVPREFIX%CS:SB:MyBlock` and you get 1 back
* Confirm that you can do `caput %MYPVPREFIX%CS:SB:MYBLOCK:SP 3` and `caget %MYPVPREFIX%CS:SB:MYBLOCK` and you get 3 back
* Confirm that you can do `caput %MYPVPREFIX%CS:SB:MyBlock:RC:ENABLE 1` and run control turns on
* Confirm that you can do `caput %MYPVPREFIX%CS:SB:MYBLOCK:RC:ENABLE 0` and run control turns off
* Confirm that all new unit tests pass and provide good coverage

Note: New documentation at https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/BlockServer#what-it-does

---

#### Code Review


### Functional Tests


### Final steps
